doc/warnings

```diff
@@ -0,0 +1,13 @@
+warning: source ../src is not a readable file or directory... skipping.
```

Comment:
please don't check in this file, instead add it to the .gitignore.

Reply:
Let me try that.